_posts/2016-09-06-bazel-rules-protobuf.md

```diff
@@ -995,6 +994,8 @@ via the
 and
 [grpc_gateway_binary](https://github.com/pubref/rules_protobuf/tree/master/grpc_gateway#grpc_gateway_binary) rules, so you can easily bridge your gRPC apps with HTTP/1.1 gateways.
 
+Refer to the [complete list of supported languages and gRPC versions](https://github.com/pubref/rules_protobuf/#rules) for more information.
```

Comment:
Thanks for adding this, any reason why C++ is at 0.15.2 and java at pre-2?
While that is not a blocker for this blog post, I am just curious if there is anything blocking the upgrade.